### DISCUSSION TITLE
What does this error mean and how do I fix it?

### DISCUSSION THREAD

**Discussion by Hulk-Lover** [The-Wineskin-Project/WineskinServer#507] [Q&A] [answered]
I got this error when trying to run a game: `0024:err:module:fixup_imports_ilonly mscoree.dll not found, IL-only binary L"launch.exe" cannot be loaded` What is mscoree.dll, and how would I fix this?

**Comment by Gcenx** [comment, marked as answer]
That’s dll is referring to Microsoft .Net, it needs some version of .Net to run.

**Reply by Hulk-Lover** [reply]
Well, I got rid of that error
